xml/deploy_cephadm.xml

```diff
@@ -1074,23 +1099,66 @@ config: OK
 <screen>&prompt.smaster;ceph-salt apply --non-interactive</screen>
    </tip>
   </sect2>
-
-  <sect2 xml:id="day2-deployment">
-   <title>Deploying Services and Gateways</title>
-   <para>
-    After you have deployed the basic &ceph; cluster, you need to deploy core
-    services to more cluster nodes. To make the cluster data accessible to
-    clients, deploy additional services as well.
-   </para>
-   <para>
-    Currently, we support deployment of &ceph; services on the command line by
-    using the &ceph; orchestrator (<command>ceph orch</command> subcommands).
-   </para>
+  <sect2 xml:id="deploy-min-cluster-final-steps">
+    <title>Final Steps</title>
+    <para>
+      Once the <command>ceph-salt apply</command> command has completed,
+      you should have one &mon; and one &mgr; running on the
+      cluster's bootstrap minion. You should be able to run
+      the <command>ceph status</command> command successfully on any of the
+      minions that were given the <literal>admin</literal> role, either as <literal>root</literal>
+      or the <literal>cephadm</literal> user.
+    </para>
+    <para>
+      The next steps involve using the &cephadm; to deploy additional &mon;,
+      &mgr;, OSDs, the Monitoring Stack, and Gateways. These steps are
+      collectively referred to as the "post-bootstrapping phase", or sometimes
+      "Day 2 deployment".
+    </para>
+    <para>
+      Before you continue, review your new cluster's network settings. At this
+      point, the <literal>public_network</literal> setting has been populated
+      based on what was entered for <literal>/cephadm_bootstrap/mon_ip</literal>
+      in the <literal>ceph-salt</literal> configuration. However, this setting
+      was only applied to &mon;. You can review this setting with the following
+      command:
+    </para>
+<screen>&prompt.smaster;ceph config get mon public_network</screen>
+    <para>
+      This is the minimum that &ceph; requires to work, but we recommend
+      making this <literal>public_network</literal> setting <literal>global</literal>,
+      which means it will apply to all types of &ceph; daemons, and not just to MONs:
+    </para>
+<screen>&prompt.smaster;ceph config set global public_network "$(ceph config get mon public_network)"</screen>
+    <note>
+      <para>
+        This step is not required. However, if you do not use this setting, the
+        &ceph; OSDs and other daemons (except &mon;) will listen on <emphasis>all addresses</emphasis>.
+      </para>
+      <para>
+        If you want your OSDs to communicate amongst themselves using a
+        completely separate network, run the following command:
+      </para>
+<screen>&prompt.smaster;ceph config set global cluster_network "<replaceable>cluster_network_in_cidr_notation</replaceable>"</screen>
```

Comment:
Wouldn't this be `ceph config set osd cluster_network` like https://susedoc.github.io/doc-ses/master/single-html/ses-deployment/#storage-bp-net-private?

Reply:
Upstream recommends `global`. I'm pretty sure both `public_network` and `cluster_network` should be set globally, not just for a particular type of daemon.

https://docs.ceph.com/en/latest/rados/configuration/network-config-ref/#cluster-network

Cluster Network

If you declare a cluster network, OSDs will route heartbeat, object replication and recovery traffic over the cluster network. This may improve performance compared to using a single network. To configure a cluster network, add the following option to the [global] section of your Ceph configuration file.

```
[global]
        # ... elided configuration
        cluster network = {cluster-network/netmask}
```

Reply:
We go with upstream for now, unless we need someone else to verify this?